packages/frontend/docs/Linting-and-formatting.md

```diff
@@ -0,0 +1,60 @@
+# Linting and Formatting
```

Comment:
<3 thanks for the documentation! This is great stuff.

I almost feel bad asking to have Prettier removed given how well it's described here :)